docs/contributors_guide/Contribution_Procedure.rst

```diff
@@ -55,53 +113,67 @@ Contribution Procedure
      * Use ``git rebase -i`` to squash commits (use ``git rebase -i HEAD~n`` to
        go back ``n`` commits)
 
+   * Make sure to select the option `Allow edits from maintainers`.  This
+     will allow people in the organization to make edits to the pull request.
```

Comment:
Some people might not want others editing their commits. In that case, it's worth saying something like, "or if you don't want us messing with your commit, be attentive to requests for changes."